.github/workflows/call-lint-chart.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
       # https://github.com/helm/chart-testing-action
       # Pre-requisites: A GitHub repo containing a directory with your Helm charts (e.g: charts)
       - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.4.0
+        uses: helm/chart-testing-action@v2.3.1
         with:
           # CT version
           version: v3.6.0
```

Comment:
This code patch appears to be a version downgrade of the "helm/chart-testing-action" action used in a GitHub Actions job. 

Given that reducing the version number may entail missing out on features, it would be helpful to have an explanation as to why this change is necessary. Ideally, this should be expressed either in comments in the code or in a related documentation.

Aside from that, there are no evident bugs or other issues with the code patch provided.